#src/index.html

```diff
@@ -49,48 +49,90 @@
                     </ul>
                 </nav>
             </div>
-            <div class="header__row row">
-                <div class="header__hero hero col-md-12">
-                    <h1 class="hero__header">Explore and Travel</h1>
-                    <div class="hero__form form">
-                        <h4 class="form__title">Holiday finder <br>____</h4>
-                        <label for="locations" class="form__label label">Location</label>
-                        <select name="locations" id="locations" class="form__select">
-                            <option value="China" class="form__option">China</option>
-                            <option value="" class="form__option"></option>
-                            <option value="" class="form__option"></option>
-                            <option value="" class="form__option"></option>
-                        </select>
-                        <label for="activities" class="form__label label">Activity</label>
-                        <select name="activities" id="activities" class="form__select">
-                            <option value="surfing" class="form__option option">Surfing</option>
-                            <option value="" class="form__option"></option>
-                            <option value="" class="form__option"></option>
-                            <option value="" class="form__option"></option>
-                        </select>
-                        <label for="grade" class="form__label label">Grade</label>
-                        <select name="grade" id="grade" class="form__select">
-                            <option value="Losem" class="form__option">Losem</option>
-                            <option value="" class="form__option"></option>
-                            <option value="" class="form__option"></option>
-                            <option value="" class="form__option"></option>
-                        </select>
-                        <label for="date" class="form__label label">Date</label>
-                        <select name="date" id="date" class="form__select">
-                            <option value="January" class="form__option">January</option>
-                            <option value="" class="form__option"></option>
-                            <option value="" class="form__option"></option>
-                            <option value="" class="form__option"></option>
-                        </select>
-                        <button class="hero__button button">Explore</button>
+        </div>
+    </header>
+    <main>
+        <div class="main__hero hero col-md-12">
+            <div class="hero__container container container--wider">
+                <div class="hero__row row">
+                    <div class="hero__form form col-md-5">
+                        <h1 class="hero__header">Explore and<br>Travel</h1>
```

Comment:
header is some single block which is always on top of the webpage
this is not header of the page, this is Heading of the text

Also, there should always be **only one H1 tag per page** and it is usually a logo or website's name on header.
Everything else should use h2 and h3's depending on level of nesting.

```
header
  h1
main
  block
    h2.block__heading
    block__info
      h3.block__subheading
```